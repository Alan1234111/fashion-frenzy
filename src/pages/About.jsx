import {Link} from "react-router-dom";
import "bootstrap/dist/css/bootstrap.min.css";
import Nav from "react-bootstrap/Nav";
import githubMark from "../img/github-mark.svg";

function About() {
  return (
    <div data-testid="about" id="about" className="d-flex flex-column justify-content-center align-items-center p-5 fw-bold fs-5 text-center">
      <p>This is a fictional store and none of the products displayed here exist.</p>
      <p>Products' information and images: Fake Store API.</p>
      <Nav.Link as={Link} to="https://github.com/Alan1234111" target="_blank" className="d-flex align-items-center fs-5 fw-normal">
        Alan &#169; <img className="h-25 mx-2" src={githubMark} alt="" />
      </Nav.Link>
    </div>
  );
}

export default About;
