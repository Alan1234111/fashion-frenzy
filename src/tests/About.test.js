import React from "react";
import {render, screen} from "@testing-library/react";
import {MemoryRouter} from "react-router-dom";
import About from "../pages/About";

describe("About", () => {
  it("matches the snapshot", () => {
    render(
      <MemoryRouter>
        <About />
      </MemoryRouter>
    );
    expect(screen.getByTestId("about")).toMatchSnapshot();
  });
});
