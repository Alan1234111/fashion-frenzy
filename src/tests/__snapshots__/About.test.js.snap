// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About matches the snapshot 1`] = `
<div
  class="d-flex flex-column justify-content-center align-items-center p-5 fw-bold fs-5 text-center"
  data-testid="about"
  id="about"
>
  <p>
    This is a fictional store and none of the products displayed here exist.
  </p>
  <p>
    Products' information and images: Fake Store API.
  </p>
  <a
    class="d-flex align-items-center fs-5 fw-normal nav-link"
    href="https://github.com/Alan1234111"
    target="_blank"
  >
    Alan Â© 
    <img
      alt=""
      class="h-25 mx-2"
      src="github-mark.svg"
    />
  </a>
</div>
`;
